src/app.js

```
@@ -1 +1,82 @@
+/* eslint-disable no-console */
 // write code here
+'use strict';
+
+const fs = require('fs/promises');
+const path = require('path');
+
+async function move(oldPath, newPath) {
+  try {
+    if (!oldPath || !newPath) {
+      throw new Error('Exactly two arguments are required');
+    }
+
```

Critical: The move function wraps its entire body in a try/catch and the catch only logs the error. This swallows errors and prevents the CLI wrapper from receiving a rejected promise. Remove the top-level try/catch here (or rethrow the error after logging) so callers can handle errors and the process can exit with non-zero. This is required so tests and the CLI detect failures (see checklist items about exiting non-zero on errors).